core/src/services/azdls/core.rs

```diff
@@ -318,3 +326,10 @@ impl AzdlsCore {
         }
     }
 }
+
+fn create_continuation(start_after: &str) -> String {
```

Comment:
How about naming it as `generate_continuation_from_start_after`? Verbose, but more clear.